packages/mgt/src/components/mgt-teams-channel-picker/mgt-teams-channel-picker.scss

```diff
@@ -7,28 +7,10 @@
 
 @import '../../../../../node_modules/office-ui-fabric-core/dist/sass/References';
 @import '../../styles/shared-styles.scss';
+@import '../../styles/theme.scss';
```

Comment:
Would this import the entire theme for all components  (and in the people picker scss file)? If so, consider separating the things you need from the theme.scss file to a separate file and import that instead of importing everything.